apps/studio/components/interfaces/Database/Publications/PublicationsTables.tsx

```diff
@@ -102,7 +102,7 @@ export const PublicationsTables = () => {
                 <TableRow>
                   <TableHead>Name</TableHead>
                   <TableHead>Schema</TableHead>
-                  <TableHead>Description</TableHead>
+                  <TableHead className="hidden lg:table-cell">Description</TableHead>
```

Comment:
This header would linger when its above related TableCell would be removed on smaller breakpoints. Now they are 1:1.